Task1_1/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Threading.Tasks.Dataflow;
+
+namespace Task1_1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * Assuming the input has no mistakes, that means:
+             * 1. User input two numbers
+             * 2. Number b is greater that number a
+             * 3. User input positive numbers
+             */
+
+            Console.WriteLine("Input first number: (a)");
+            var inputA = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("\nInput second number (b)");
+            var inputB = int.Parse(Console.ReadLine());
+
+            var output = FindDoublesAsNumbers(inputA, inputB);
+            Console.WriteLine("\nOutput: ");
+            foreach (var number in output)
+            {
+                Console.WriteLine(number);
+            }
+        }
+
+
```

Comment:
Cool but to now it is supposed that that you don't know about List<T> collection, try not use it )))

Reply:
Good Morning @IgorKheidorov 

Suggestion implemented. Refactored return type from List<int> to int[]